tox.ini

```
@@ -1,8 +1,10 @@
 [tox]
-envlist = py312, coverage, pylint
+envlist = py311-django{42,52}, py312-django{42,52}, coverage, pylint
```

```suggestion
envlist = py{311,312}-django{42,52}, coverage, pylint
```

This is equivalent and much less repetitive.